@ecomplus/storefront-app/src/components/html/EcOrderInfo.html

```diff
@@ -185,14 +202,46 @@
                         <div
                           v-for="shippingLine in localOrder.shipping_lines"
                           :key="shippingLine._id"
+                          class="order-info__shipping-freight"
                         >
                           <div class="text-muted">
                             <i class="fas fa-truck mr-1"></i>
                             {{ i19freight }}
                           </div>
+                          <span v-if="shippingLine.app">
+                            {{ shippingLine.app.label }}
+                          </span>
                           <shipping-line :shipping-line="shippingLine"/>
+
+                          <div
+                            v-if="shippingLine.tracking_codes"
+                            class="order-info__shipping-tracking"
+                          >
+                            <div class="text-muted">
+                              <i class="fas fa-search mr-1"></i>
+                                {{ i19deliveryTrack }}
+                            </div>
+                            <div
+                              v-for="tracking in shippingLine.tracking_codes"
+                            >
+                              <strong>
+                                <a
+                                  v-if="tracking.link"
+                                  href="`${tracking.link}`"
```

Comment:
```suggestion
                                  :href="`${tracking.link}`"
```

Faltou um `:` (ou `v-bind`) aqui...